R/fable_model.R

```diff
@@ -7,61 +7,77 @@
 #'
 #' @param model A `fable` model object. For  models that use a formula interface time
 #' can be accessed using `time`.
+#' @param bias_correction logical. If TRUE, bias will be corrected based on
+#' the fitted values. This is only a temporary solution and will be replaced
+#' as soon as possible.
 #' @inheritParams bsts_model
 #' @return A dataframe of predictions (with columns representing the
 #'  time horizon and rows representing samples).
 #' @export
 #' @importFrom tsibble tsibble
 #' @importFrom fabletools model forecast
 #' @importFrom purrr map
-#' @importFrom dplyr bind_rows
+#' @importFrom dplyr bind_cols
 #' @examples
 #' ## Used on its own
 #' fable_model(y = EpiSoon::example_obs_rts[1:10, ]$rt,
 #'            model = fable::ARIMA(y ~ time),
```

Comment:
please add as a second example - this is not likely to be the default option as it feels a bit hacky